seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/sink/HiveSinkFactory.java

```diff
@@ -66,7 +74,8 @@ public OptionRule optionRule() {
 
         ReadonlyConfig finalReadonlyConfig =
                 generateCurrentReadonlyConfig(readonlyConfig, catalogTable);
-        return () -> new HiveSink(finalReadonlyConfig, catalogTable);
+        CatalogTable finalCatalog = renameCatalogTable(finalReadonlyConfig, catalogTable);
```

Comment:
replace with target hive sink table name, if not replace here, will pass source table name to hive.
like `fake` to hive sink. so when use this catalog, will has issue, replaced here